### ISSUE TITLE
NTTのネットワークでの使用について (Usage on NTT's network)

### ISSUE THREAD

**Issue by ShuttleTydirium** [YuukiJapanTech/CA8271x#15]
お世話になります。

NTT 10G-EPON <M>C ONU をNTT（ISPはen光）のネットワークで使用していますが、SFP-ONUに切り替えたいと考えています。このGitHubリポジトリの情報を参考にすれば、NTTのONUから必要な設定を抽出し、Taobaoの「CIG XE-99S」SFP-ONUに交換して、NTTのネットワークに接続して使用できるでしょうか？

よろしくお願いします。

_I'm currently using an NTT 10G-EPON <M>C ONU on NTT's network (ISP is en-hikari). I want to switch to an SFP-ONU. If I follow the information in this GitHub repository, will I be able to extract the necessary settings from the NTT ONU, replace it with a 'CIG XE-99S' SFP-ONU purchased from Taobao, and successfully connect and use it on NTT's network?_

**Comment by YuukiJapanTech** [comment]
No.
NTT's ONU is currently under analysis and cannot be fully replaced.

Also, the method of extracting information from NTT ONU ( NTT ONU hack ) is not described in this repository.
Also, we do not plan to include this information at this time.

**Comment by ShuttleTydirium** [comment]
Thank you for your reply.

`NTT's ONU is currently under analysis and cannot be fully replaced.`
Will you share updates here?

Where can I find NTT ONU hack information, or maybe it will be shared after NTT ONU can be replaced?

It is very unfortunate NTT only has SFP ONU for 1G connection. Not 10G connection. I do not like 10GBaseT in my network. It is too hot and uses a lot of power.

I see that older 10G-EPON **B** ONU has SFP module inside. Is this SFP module possible to use directly inside router? Are there any other options to replace NTT 10GBaseT ONU?

Thank you

**Comment by ShuttleTydirium** [comment]
Yes, of course, I don't want to share the unique ID numbers of the device. 
I understand the difference between ONU and transceivers. I have programmed embedded hardware before, but I have not modified an ONU before. Thank you for your comment.

I saw on a twitter post you made a comment about "prerelease" NTT 10G SFP-ONU. Do you know if NTT plans to release an official 10G SFP-ONU soon? I would be happy to buy one instead of modifying one. But either way I want an SFP-ONU. :)

Thank you.

**Comment by missing233** [comment]
It doesn't seem possible at the moment, unless you have the ability to get the SDK and do some custom development.

**Comment by YuukiJapanTech** [comment]
issueではないのでクローズ
